### ISSUE TITLE
Footer is not regular style

### ISSUE THREAD

**Issue by Atiladanvi** [adenvt/laravel-coreui-vue#15]
Please add the follow classes into app.blade.php to footer work.
**Now:**
![image](https://user-images.githubusercontent.com/11599205/52367554-212bf500-2a2b-11e9-9f9f-d641d8c167fd.png)
```html 
<body class="app header-fixed sidebar-fixed sidebar-lg-show pace-done pace-done">
```
**After:**
![image](https://user-images.githubusercontent.com/11599205/52367613-3e60c380-2a2b-11e9-9b03-1f8e96a3bd8e.png)

**Comment by adenvt** [comment]
I have checked and found Core-UI has 2 styles, sidebar fixed and non fixed. footer on non-fixed will be like first one and fixed sidebar like second one. I plan to make this style configurable on next release.

**Comment by adenvt** [comment]
Fixed in 1.1.0
